recipes/magicblast/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.4.0" %}
+
+package:
+  name: magicblast
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/magicblast/LATEST/ncbi-magicblast-{{ version }}-x64-linux.tar.gz # [linux]
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/magicblast/LATEST/ncbi-magicblast-{{ version }}-x64-macosx.tar.gz # [macos]
+  sha256: 9995524a2ce67850516debb3341f268d485113f1c28005e39b5fbf47fe27d250 # [linux]
+  sha256: f48daeb86065be4f256820b5c189f0141eafdf50e5a0b8e837b381e82a447b0e # [macos]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - patchelf
```

Comment:
Hmm. These are not needed if you just copy the binaries. If you can at all, please do try to build from source though!

Reply:
I thought I removed them. Sorry. Removed now.